### DISCUSSION TITLE
How to get the balance of TRON?

### DISCUSSION THREAD

**Discussion by AlleniCode** [trustwallet/wallet-core#2296] [Q&A] [answered]
Hi, is there a sample of how to get the balance of TRON tokens? Thank you very much!

**Comment by optout21** [comment, marked as answer]
Wallet core does not deal with communication with nodes/network.
Balance can be retrieved using a node, through API/RPC.
Maybe a good starting point is this:
https://tronprotocol.github.io/documentation-en/api/http/#api-index

**Reply by AlleniCode** [reply]
Thanks！
https://developers.tron.network/reference/account-getaccount
